appengine/endpoints-v1-using-endpoints-frameworks-v2/Jenkins.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Wow - what a long path `endpoints-v1-using-endpoints-frameworks-v2`.  Seems quite wordy, but Ok if you don't want to use abbreviations.